spacialgaze-plugins/SG.js

```diff
@@ -91,19 +91,18 @@ SG.reloadCSS = function () {
 };
 
 //Daily Rewards System for SpacialGaze by Lord Haji
+//Daily Rewards System for SpacialGaze by Lord Haji
 SG.giveDailyReward = function (userid, user) {
 	if (!user || !userid) return false;
 	userid = toId(userid);
-	if (!user.named) return false;
-	if (!user.autoconfirmed) return false;
```

Comment:
- The user must be named to get a reward

- The user must be autoconfirmed to prevent stardust farming

- Could you set this up so that a use must wait 24 hours after their first connection to SG to get their first reward? This will prevent stardust farming too.